justfile

```diff
@@ -47,13 +47,13 @@ release *ARGS:
 test:
     cargo test --no-fail-fast --all-features --all-targets
 
-integration-test:
-    #!/usr/bin/env bash
-    ci/build_scripts/build.sh x86_64-unknown-linux-musl
```

Comment:
Don't we need to build the binaries?

Reply:
Yes we still need to build the binaries, but that was part of the refactoring, it is doing using the existing `release` recipe. For example:

```
integration-test *ARGS: release
```